definitions/pipelines/rnaseq_star_fusion_with_xenosplit.cwl

```diff
@@ -0,0 +1,246 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "STAR-RNA-Seq alignment and transcript/gene abundance workflow with Xenosplit"
+requirements:
+    - class: MultipleInputFeatureRequirement
+    - class: SubworkflowFeatureRequirement
+    - class: ScatterFeatureRequirement
+    - class: StepInputExpressionRequirement
+inputs:
+    instrument_data_bams:
+        type: File[]
+    outsam_attrrg_line:
+        type: string[]
+    graft_star_genome_dir:
+        type: Directory
+        doc: "Location of the STAR human reference directory"
```

Comment:
suggested change:
doc: "Location of the STAR reference for the graft species (usually human)"